src/base.css

```diff
@@ -1344,6 +1346,21 @@ Possible extra rowspan handling
 		font-weight: bold;
 	}
 
+/** Character examples (from the I18N template) *******************************/
+.codepoint bdi {
+    line-height: 1em;
+    font-size: 120%;
```

Comment:
I don't think the inline font-size change is great. It interferes with understanding the relationship of the glyph to the rest of the text. E.g. for é it makes it look like maybe this is a special lowercase e that is the same height as the capital letters (for some weird Unicode-y reason).

Reply:
Btw, if you want to zoom on :hover, I think the code that works is

```
bdi { display: inline-block; }
bdi:hover { transform-origin: center; scale: 150%; }
```

Reply:
Let me try the :hover again. I think I was missing the inline-block part.